### DISCUSSION TITLE
Definicion del Proyecto Imposible

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-EM2022/Grupo6#141] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by JorEl057** [comment]
Chicos, que tal como estan :

Manejo de Control de versiones: Ok
Manejo de Estructura de proyecto: Ok
Presentación Final: Falta presentar video con la demostración de lo hecho, la presentación grupal, subir código.

**Comment by JorEl057** [comment]
Devolución Grupal: @mariogonzalezispc  @lilenguzman01  @Maxilopez28  @TerraWolf  @enriqueripoli 
Nota Final Grupal: 9 (Nueve)

Ideación del Proyecto: 9 (Nueve)
Presentación: 8 (Ocho)
Uso de recursos: 10 (Diez)
Desarrollo: 9 (Nueve)
Implementación: 9 (Nueve)

Nota Final Grupal: 9 (Nueve)

Estimados tomaron la idea de Hidroponía como proyecto y la ampliaron como un proyecto general de inteligencia y comunicación entre los sensores para cultivos hidropónicos, aplicable en cualquier sistema de cultivos de este tipo. Esto demuestra su compromiso y RSU. En su presentación que cumplieron correctamente con un tiempo acorde, hablan sobre la necesidad en forma general y la implementación en forma particular. Trabajaron demostrando una organización y de recursos agiles muy bien logrados. Finalmente el desarrollo y la implementación del proyecto también fueron muy logrados. Felicitaciones y continúen trabajando así!!!
